bindings/protocols/coap/index.html

```diff
@@ -1,181 +1,464 @@
+            <h3>BlockWiseTransferParameters</h3>
+            <table class="def">
+                <thead>
+                    <tr>
+                        <th>Vocabulary term</th>
+                        <th>Description</th>
+                        <th>Assignment</th>
+                        <th>Type</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td><code>coap:block2SZX</code></td>
+                        <td></td>
+                        <td>optional</td>
+                        <td><a href="http://www.w3.org/2001/XMLSchema#integer">integer</a></td>
+                    </tr>
+                    <tr>
+                        <td><code>coap:block1SZX</code></td>
+                        <td></td>
+                        <td>optional</td>
+                        <td><a href="http://www.w3.org/2001/XMLSchema#integer">integer</a></td>
+                    </tr>
+                </tbody>
+            </table>
+        </section>
+        <section id="QuickBlockWiseTransferParameters">
+            <h3>QuickBlockWiseTransferParameters</h3>
+            <table class="def">
+                <thead>
+                    <tr>
+                        <th>Vocabulary term</th>
+                        <th>Description</th>
+                        <th>Assignment</th>
+                        <th>Type</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td><code>coap:qblock2SZX</code></td>
+                        <td></td>
+                        <td>optional</td>
+                        <td><a href="http://www.w3.org/2001/XMLSchema#integer">integer</a></td>
+                    </tr>
+                    <tr>
+                        <td><code>coap:qblock1SZX</code></td>
+                        <td></td>
+                        <td>optional</td>
+                        <td><a href="http://www.w3.org/2001/XMLSchema#integer">integer</a></td>
+                    </tr>
+                </tbody>
+            </table>
+        </section>
+    </section>
+    <section id="mappings">
+        <h2>Mappings</h2>
+        <p>
+            This section describes the strategies and default values to use
+            protocol specific concepts within the
+            <a href="https://w3c.github.io/wot-architecture/#dfn-interaction-model">WoT Interaction model</a>.
+        </p>
+        <section id="default-mappings">
+            <h3>Default mappings</h3>
+            <table class="def">
+                <thead>
+                    <tr>
+                        <th>Operation</th>
+                        <th>Default Binding</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td><code>td:readProperty</code></td>
+                        <td><code>"coap:method": "coap:get"</code></td>
```

Comment:
Should this be `<td><code>"coap:method": "get"</code></td>`?

Reply:
In the HTTP binding and the old CoAP binding, the method names are unprefixed and all upper case, i. e.:

```suggestion
                        <td><code>"coap:method": "GET"</code></td>
```

Reply:
Or would it make sense to also prefix the method names from an RDF standpoint?

Reply:
Yes, the prefix can be omitted. The reason for using capital letters is that the HTTP in RDF used these: https://www.w3.org/TR/HTTP-in-RDF10/#example